nixos/modules/services/web-servers/webhook.nix

```
@@ -0,0 +1,89 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.webhook;
+  webhook = pkgs.webhook;
+  user = if (cfg.user == null) then "webhook" else cfg.user;
+  group = if (cfg.group == null) then "webhook" else cfg.group;
+  settingsFormat = pkgs.formats.json {};
+  configFile = settingsFormat.generate "webhook.json" cfg.settings;
+
+in {
+  options.services.webhook = {
+
+    enable = mkEnableOption "the webhook service";
+
+    user = mkOption {
+      default = null;
+      description = "User webhook service runs as. If null, it will default to `webhook`";
```

description sadly aren't markdown (yet), but docbook.
```suggestion
      description = ''
        User the webhook service runs as. If <literal>null</literal>,
        it will create and use a user named <literal>"webhook"</literal>.
      '';
```
also applies to `group`